docs/src/tutorial/add-remove-nodes.md

```diff
@@ -0,0 +1,116 @@
+# Adding and Removing Nodes
+
+## What you will need
+- Multipass (See [Multipass Installation](https://multipass.run/install))
+- A bootstrapped cluster using Canonical K8s (See (getting-started.md)[Getting Started]
+- The `dns` and `network` components enabled on your cluster.
+
+### 1. Verify the Status Of Your Cluster
+
+Verify the status of your cluster with:
+
+```
+sudo k8s status
+```
+
+Make sure you can see `k8s is ready.` in the output of the command.
+
+### 2. Start a Multipass VM
+
+Our second node will live on Ubuntu 22.04 on a UMultipass VM. Let's launch it:
+
+```
+multipass launch --name expert-elf
```

Comment:
```suggestion
multipass launch 22.04 --name expert-elf
```
As we explicitly claim a 22.04 image I think we should make sure we explicitly get one. In a few months they'll be getting 24.04 instead